functions/Set-DbaMaxDop.ps1

```diff
@@ -104,7 +104,7 @@ function Set-DbaMaxDop {
     }
 
     process {
-        if (Test-Bound -ParameterName Database, AllDatabases, ExcludeDatabase) {
+        if (Test-Bound -Not -Min 0 -Max 1 -ParameterName Database, AllDatabases, ExcludeDatabase) {
```

Comment:
Works for me, but could reduce `-Not -Min 0 -Max 1` to `-Min 2`

Reply:
Oh yes thanks, just pushed an update.